### PR TITLE
Remove Media references

### PR DESCRIPTION
**Pull request by franmomu**
[Media support was removed in master](https://github.com/sonata-project/SonataClassificationBundle/pull/196), this removes some references added in a merge.

### CODE REVIEW

File: src/Model/CollectionInterface.php [hunk 1]
```diff
@@ -90,15 +90,6 @@ public function setUpdatedAt(\DateTime $updatedAt);
...
     public function getUpdatedAt();
 
-    public function setMedia(?MediaInterface $media = null);
-
-    /**
-     * @return MediaInterface
-     */
-    public function getMedia();
-
-    public function setContext(ContextInterface $context);
```
Comment (jordisala1991) [inline src/Model/CollectionInterface.php:100]: this removal is correct?

Comment (franmomu) [reply src/Model/CollectionInterface.php:100]: Not at all, thanks.

**Review by jordisala1991 — DISMISSED** [review]

**Comment by jordisala1991** [comment]
I cant merge this PR without first merging 3.x into master

**Review by core23 — DISMISSED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Thank you @franmomu

**Pull request merged**